loops/csharp/code.cs

```diff
@@ -0,0 +1,16 @@
+﻿var u = int.Parse(args[0]);
+var r = Random.Shared.Next(10_000);
+
+//Use span instead of array
+Span<int> a = new int[10_000];
+
+//using a.Length instead of 10_000 constant should give the compiler better optimization hints
+for (var i = 0; i < a.Length; i++)
+{
+    for (var j = 0; j < 100_000; j++)
+    {
+        a[i] = a[i] + j % u;
```

Comment:
Nit (because it won't affect perf), but any reason why his isn't using a compound assignment? It could do it in the C++ version as well. Anyway `%` has higher precedence.

```suggestion
        a[i] += j % u;
```

Reply:
Thanks for pointing this out, it is a nitpick, but the code can still look better :)